Dockerfile

```diff
@@ -31,7 +31,12 @@ COPY tiles /usr/share/nginx/html/tiles
 COPY --from=builder /pbf /usr/share/nginx/html/tiles/pbf
 COPY template.lua /usr/local/openresty/site/lualib/resty/template.lua
 
-COPY nginx.conf /usr/local/openresty/nginx/conf/nginx.conf
+ARG BUILD_DATE
+ENV image_build_date=[Build_$BUILD_DATE]
+
+COPY nginx.conf /etc/nginx/templates/nginx.conf.template
```

Comment:
rename `nginx.conf` to `nginx.conf.template` also in the code, so it is clear that this is indeed a template

Reply:
Changed the name